spring-cassandra/src/main/java/org/baeldung/springcassandra/model/Person.java

```diff
@@ -0,0 +1,98 @@
+package org.baeldung.springcassandra.model;
+
+import org.springframework.data.cassandra.core.mapping.PrimaryKey;
+import org.springframework.data.cassandra.core.mapping.Table;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.Date;
+import java.util.Objects;
+import java.util.UUID;
+
+@Table
+public class Person {
+
+    @PrimaryKey
+    private UUID id;
+
+    private String firstName;
+
+    private String lastName;
+
+    private LocalDate birthDate;
+
+    private LocalDateTime lastVisitedDate;
+
+    private Date lastPurchasedDate;
+
+    public Person(UUID id, String firstName, String lastName) {
+        this.id = id;
+        this.firstName = firstName;
+        this.lastName = lastName;
+    }
+
+    public UUID getId() {
+        return id;
+    }
+
+    public void setId(UUID id) {
+        this.id = id;
+    }
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public void setFirstName(String firstName) {
+        this.firstName = firstName;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+
+    public void setLastName(String lastName) {
+        this.lastName = lastName;
+    }
+
+    public LocalDate getBirthDate() {
+        return birthDate;
+    }
+
+    public void setBirthDate(LocalDate birthDate) {
+        this.birthDate = birthDate;
+    }
+
+    public LocalDateTime getLastVisitedDate() {
+        return lastVisitedDate;
+    }
+
+    public void setLastVisitedDate(LocalDateTime lastVisitedDate) {
+        this.lastVisitedDate = lastVisitedDate;
+    }
+
+    public Date getLastPurchasedDate() {
+        return lastPurchasedDate;
+    }
+
+    public void setLastPurchasedDate(Date lastPurchasedDate) {
+        this.lastPurchasedDate = lastPurchasedDate;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o)
+            return true;
```

Comment:
Let put in the `{}` here - I know IDEs miss them out, but it's a better standard to include them.